testing/generate.js

```diff
@@ -117,14 +120,14 @@ export class RetoolApplication {
   }
 
   async openEditor() {
-    await this.page.setDefaultTimeout(60000)
+    await this.page.setDefaultTimeout(0)
     let url = ''
     if (this.folder) {
       url = 'http://${HOSTNAME}:3000/editor/'+this.folder+'/'+this.name
     } else {
       url = 'http://${HOSTNAME}:3000/editor/'+this.name
     }
-    await this.page.goto(url, {waitUntil: 'load', timeout: 0})
+    await this.page.goto(url, {waitUntil: 'load', timeout: 600000})
```

Comment:
maybe this timeout should be 0? since we confirmed that means disabled.

Reply:
0 was not working as expected; safer to set timeout to a large amount